tests/runtime/probe

```diff
@@ -137,7 +137,8 @@ AFTER ./testprogs/syscall read
 
 NAME kprobe_module_missing
 PROG kprobe:nonsense:vfs_read { printf("SUCCESS %d\n", pid); exit(); }
-EXPECT ERROR: Error attaching probe: kprobe:nonsense:vfs_read: specified module nonsense in probe kprobe:nonsense:vfs_read is not loaded.
+EXPECT WARNING: specified module nonsense in probe kprobe:nonsense:vfs_read is not loaded.
+EXPECT ERROR: Attachment failed for all (non-special) probes.
```

Comment:
`(non-special)` is weird, maybe we want to just call these "real probes"? "actual probes"? Worthwhile to standardize on a name here and in docs, possibly.

Reply:
Yeah "non-special" is crappy. Let me marinate on some better wording. I could also just remove this as an error. If only `BEGIN` attaches - oh well. At least we told the user we couldn't attach these other ones.

Reply:
Just removed this special case.